src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -137,7 +136,8 @@ interface SearchBarDropdownProps {
 export const SearchBarDropdown = (props: SearchBarDropdownProps) => {
   const { isLoading } = props
   const { chainId } = useWeb3React()
-  const showChainComingSoonBadge = chainId && BACKEND_NOT_YET_SUPPORTED_CHAIN_IDS.includes(chainId) && !isLoading
+  const COMING_SOON_CHAINS = new Set([ChainId.BNB, ChainId.AVALANCHE])
```

Comment:
BASE should also show the coming soon badge

Reply:
Base is supported

<img width="533" alt="image" src="https://github.com/Uniswap/interface/assets/5773490/0ab07599-34de-43d9-87d5-c45b0003437d">